src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2455,18 +3070,7 @@ def update_site_configs(cmd, resource_group_name, name, slot=None, number_of_wor
         if max_replicas is not None:
             setattr(configs, 'function_app_scale_limit', max_replicas)
         return update_configuration_polling(cmd, resource_group_name, name, slot, configs)
-    try:
-        return _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'update_configuration', slot, configs)
-    except Exception as ex:  # pylint: disable=broad-exception-caught
```

Comment:
Why the removal of this exception handler block?

Reply:
For context, this exception handling was added as part of this commit: https://github.com/Azure/azure-cli/commit/ef54889112c5ff4009ea653fdbf91f4fe88a8abf

And this function is run with the `az webapp config set` command. 

I do not think it is right that we are catching Conflict and Bad Request exceptions. I think that if either of those two exceptions are simply logged instead of thrown back to the client when run via a script in a pipeline, customers might miss that there were issues with updating the site config. Even for the purposes of this PR, the caller of this function should know via a thrown exception that something went wrong, and the caller should decide how to handle such exception.